package.json

```
@@ -1,68 +1,59 @@
+  "version": "2.0.0-dev",
   "private": true,
-  "description": "A dedicated open-source Discord music bot for Zhycorp based on Jukebox, easy to use, and with no coding required.",
-  "homepage": "https://github.com/zhycorp/disc-11#readme",
-  "bugs": {
-    "url": "https://github.com/zhycorp/disc-11/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/zhycorp/disc-11.git"
-  },
-  "license": "AGPL-3.0",
-  "author": "Zhycorp <support@zhycorp.net> (https://github.com/zhycorp)",
-  "main": "dist/main.js",
+  "description": "A dedicated open-source Discord bot for Zhycorp based from our Discord bot template with more features. Easy to use, and with no coding required.",
+  "repository": "https://github.com/zhycorp/disc-11",
+  "license": "BSD-3-Clause",
+  "author": "Zhycorp <support@zhycorp.net>",
+  "main": "index.js",
   "scripts": {
     "build": "npm run lint && npm run compile",
     "compile": "tsc --build tsconfig.json",
-    "lint": "eslint . --ignore-path .gitignore --ext .ts",
-    "lint:fix": "eslint . --ignore-path .gitignore --ext .ts --fix",
-    "start": "node start.js",
-    "start:dev": "echo [INFO] Sharding disabled when using ts-node && ts-node src/bot"
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "start": "node .",
+    "start:dev": "ts-node -r dotenv/config src/bot.ts"
   },
   "eslintConfig": {
-    "extends": "@zhycorp/eslint-config/typescript",
-    "ignorePatterns": "dist/*"
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    },
+    "extends": "@zhycorp/eslint-config/typescript"
   },
   "dependencies": {
-    "discord.js": "^12.5.3",
-    "dotenv": "^10.0.0",
+    "@discordjs/voice": "^0.6.0",
+    "date-fns": "^2.23.0",
+    "discord.js": "^13.1.0",
     "entities": "^3.0.1",
     "ffmpeg-static": "^4.4.0",
-    "iso8601-duration": "^1.3.0",
-    "node-fetch": "^2.6.1",
-    "opusscript": "^0.0.8",
-    "parse-ms": "^2.1.0",
-    "pretty-ms": "^7.0.1",
-    "semver": "^7.3.5",
+    "got": "^11.8.2",
+    "soundcloud.ts": "^0.4.1",
+    "spotify-url-info": "^2.2.3",
     "tslib": "^2.3.1",
     "winston": "^3.3.3",
-    "ytdl-core": "^4.9.1",
-    "ytpl": "^2.2.3",
-    "ytsr": "^3.5.3"
+    "youtube-dl-exec": "^1.2.6",
+    "youtubei": "^0.0.1-rc.23"
   },
   "devDependencies": {
-    "@types/node": "^16.9.2",
-    "@types/node-fetch": "^2.5.12",
-    "@types/semver": "^7.3.8",
-    "@typescript-eslint/eslint-plugin": "^4.31.1",
-    "@typescript-eslint/parser": "^4.31.1",
+    "@types/node": "^16.9.6",
+    "@typescript-eslint/eslint-plugin": "^4.31.2",
+    "@typescript-eslint/parser": "^4.31.2",
     "@zhycorp/eslint-config": "^4.0.0",
+    "dotenv": "^10.0.0",
     "eslint": "^7.32.0",
     "ts-node": "^10.2.1",
     "typescript": "^4.4.3"
   },
   "optionalDependencies": {
     "@discordjs/opus": "^0.6.0",
     "bufferutil": "^4.0.3",
-    "erlpack": "github:hzmifork/erlpack",
+    "erlpack": "github:discord/erlpack",
```

```suggestion
    "erlpack": "github:hzmifork/erlpack",
```
Still no ETA until hzmi's fork merged, use hzmifork or don't add this
https://github.com/discord/erlpack/pull/41